### PR TITLE
replaced hibernate-jpa-2.1-api by javax.persistence-api

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

File: employee-rostering-gwtui/pom.xml [hunk 1]
```diff
@@ -231,15 +237,15 @@
                       java.lang.NoClassDefFoundError: javax/persistence/spi/PersistenceUnitInfo
                       The best approach seems to be to keep hibernate-jpa-2.1-api and disable Enforcer.
                       -->
-                      <dependency>
+		      <dependency>
                         <groupId>org.jboss.errai</groupId>
                         <artifactId>errai-jpa-client</artifactId>
-                        <type>jar</type>
-                        <ignoreClasses>
+			<type>jar</type> -->
```
Comment (rsynek) [inline employee-rostering-gwtui/pom.xml]: Comment closing tag without an opening tag.

File: employee-rostering-gwtui/pom.xml [hunk 2]
```diff
@@ -231,15 +237,15 @@
...
+			<ignoreClasses>
                           <ignoreClass>javax.persistence.*</ignoreClass>
-                        </ignoreClasses>
-                      </dependency>
-                    </dependencies>
+		        </ignoreClasses>
+	              </dependency>
+	            </dependencies> -->
```
Comment (rsynek) [inline employee-rostering-gwtui/pom.xml]: ditto

File: employee-rostering-gwtui/pom.xml (hunk 1 above)
Comment (mbiarnes) [reply employee-rostering-gwtui/pom.xml]: fixed

File: employee-rostering-gwtui/pom.xml (hunk 2 above)
Comment (mbiarnes) [reply employee-rostering-gwtui/pom.xml]: fixed

**Comment by mbiarnes** [comment]
Jenkins please retest this

File: employee-rostering-gwtui/pom.xml [hunk 3]
```diff
@@ -160,6 +160,12 @@
     <dependency><!-- TODO Remove me, we don't use JPA on the client, but removing this gives blank screen at runtime -->
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-jpa-client</artifactId>
+      <exclusions>
+        <exclusion>
+	  <groupId>org.hibernate.javax.persistence</groupId>
+	  <artifactId>hibernate-jpa-2.1-api</artifactId>
```
Comment (yurloc) [inline employee-rostering-gwtui/pom.xml]: Tab.

File: employee-rostering-gwtui/pom.xml [hunk 4]
```diff
@@ -221,7 +227,7 @@
               <configuration>
                 <rules>
                   <banDuplicateClasses>
-                    <dependencies combine.children="append">
+                    <dependencies combine.children="append">
```
Comment (yurloc) [inline employee-rostering-gwtui/pom.xml]: Trailing space.

**Pull request merged**